src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -6572,7 +6572,7 @@ private TupleTypeSyntax ParseTupleType()
                 var close = this.EatToken(SyntaxKind.CloseParenToken);
                 var result = _syntaxFactory.TupleType(open, list, close);
 
-                if (!result.ContainsDiagnostics && list.Count < 2)
+                if (list.Count < 2)
```

Comment:
The change here is covered by existing unit tests that were modified.

---
In reply to: [78417858](https://github.com/dotnet/roslyn/pull/13720#discussion_r78417858) [](ancestors = 78417858)